wranglerjs/index.js

```diff
@@ -3,93 +3,100 @@ const { join } = require("path");
 const { writeFileSync } = require("fs");
 const WasmMainTemplatePlugin = require("webpack/lib/wasm/WasmMainTemplatePlugin");
 
-function error(msg) {
-  console.error("Error: " + msg);
-  process.exit(1);
-  return new Error("error");
-}
-
-const rawArgs = process.argv.slice(2);
-const args = rawArgs.reduce((obj, e) => {
-  if (e.indexOf("--") === -1 && e.indexOf("=") === -1) {
-    throw error("malformed arguments");
+(async function() {
```

Comment:
Just for the record, I'm not a fan of the async iife, but I think it's the best option for now.

Reply:
Really? That's interesting! I'd love to hear your thoughts 😄

Reply:
The main difference is that it's not top-level code anymore, which hurts readability a in my opinion. Once we have Top-Level-Await support it will look a lot nicer.

There are a few technical differences too but they don't matter here, just a personal pref. I'm not blocking on it.

Reply:
![image](https://user-images.githubusercontent.com/9408157/63099869-82cd9e80-bf3b-11e9-8f66-521dccb9c519.png)
this is the first thing i see when i google 👀

Reply:
Rich meant the proposal from 2 years ago and from a bundler percpective. Check the 2019 edit.

Reply:
oh well would you look at that 😋 just saw the headline before coming in to the office - this is why it's important to read the content!